internal/output/output_result.go

```diff
@@ -162,7 +162,7 @@ const (
 	VulnTypeUnimportant                         // 2
 )
 
-const UnfixedDescription = "No fix available"
+const UnfixedDescription = "--"
```

Comment:
output_result is also used in other output format, such as HTML. so let's not change here